### ISSUE TITLE
bug: Android 4.4.4 font sizing and shifted alignment

### ISSUE THREAD

**Issue by bradmartin** [driftyco/ionic#3107]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>4.4</span>  <span ionic-webview>webview</span>

<span ionic-description>I've duplicated this in my app along with the starter projects.
I'm using a Samsung Galaxy S5 and last week I updated to Android 4.4.4 (when the update became available). Several app, including some native (IGN) have been effected by the font-sizing. It has been scaled down quite a bit. I tested this in the ionic sidemenu starter project and everything is fine on desktop Chrome but after building and installing on the GalaxyS5 several items are not aligning correctly. I opened a topic on the Ionic forum (http://forum.ionicframework.com/t/android-4-4-4-alignment-issues-with-ionic-apps/17443) a couple days ago but have not had any responses.</span>

<span is-issue-template></span>

**Comment by mhartington** [comment]
So where exactly are you seeing this?
Just tested out the tabs and sidemenu starter projects on gs5 with 4.4.4 and everything looked good

**Comment by bradmartin** [comment]
The BuzzUp app, it's referenced in the ionic forum link above. I informed the developer about it as before my update it was not doing that, I believe pre 4.4.4 the icon was much larger and centered.
The other remaining images are from the starter project. The tabs are displaying the test below due to the scaling issue, this is live in an internal app and only present on gs5 4.4.4 that I know of at this time. If you look at the image of the login below I added an anchor below the login button, code is below. Maybe I'm using the wrong ionic css on the anchor tag but before the update everything was fine, now things seem slightly shifted. 

    <ion-modal-view>
     <ion-header-bar>
      <h1 class="title">Login</h1>
     <div class="buttons">
      <button class="button button-clear" ng-click="closeLogin()">Close</button>
    </div>
      </ion-header-bar>
      <ion-content>
       <form ng-submit="doLogin()">
        <div class="list">
        <label class="item item-input">
          <span class="input-label">Username</span>
          <input type="text" ng-model="loginData.username">
        </label>
        <label class="item item-input">
          <span class="input-label">Password</span>
          <input type="password" ng-model="loginData.password">
        </label>
        <label class="item">
          <button class="button button-block button-positive" type="submit">
		  <i class="icon ion-home"/>
		  Log in</button>
		  
		    <a href="tel:800.264.8580" class="icon-left button button-block button-outline button-   positive">    <i class="icon ion-ios7-telephone-outline"></i> Call Us</a>
        
		</label>
        </div>
      </form>
    </ion-content>
   </ion-modal-view>

![buzzup](https://cloud.githubusercontent.com/assets/6006148/6194023/f7484fc4-b385-11e4-9fd6-64423c06616c.JPG)
![loginup](https://cloud.githubusercontent.com/assets/6006148/6194024/f748772e-b385-11e4-8e7c-78f08eda7324.JPG)
![shiftedup](https://cloud.githubusercontent.com/assets/6006148/6194025/f748c1de-b385-11e4-89b0-6df53e5dc362.JPG)
![tabs](https://cloud.githubusercontent.com/assets/6006148/6194022/f748355c-b385-11e4-87bb-2457fcaa0f28.JPG)

**Comment by mhartington** [comment]
Can you check the view of the css you have, it should be in a commented section at the top.

**Comment by bradmartin** [comment]
Sorry, the view of the css? On the device or in the project?

**Comment by mhartington** [comment]
version* of the css

**Comment by bradmartin** [comment]
If you're referring to ionic.css file it is v1.0.0-beta.14

**Comment by mhartington** [comment]
Can you test this with the nightly builds

**Comment by bradmartin** [comment]
Testing it now

**Comment by bradmartin** [comment]
-nightly-1041 and the issue is still present with the same html code used
before.

**Comment by mhartington** [comment]
@bradmartin can you shoot me an email

mike at drifty dot com

**Comment by bradmartin** [comment]
Mike,

Apologies for not getting back to you sooner. I'm close to tracking this
down (I think). I created a new blank cordova app and started developing
using Polymer, all the layout and elements were fine. However, I wanted to
use Ionicons so I went to the ionic.css and copied over the ionicons
styling and some of the reset stylings. I think the issue is going to be in
the reset stylings, I haven't tracked it down yet and inspecting the
elements on the device doesn't indicate what exactly causes the alignment
issues but I'll update you on the progress. Someone else commented on the
issue in the Ionic forums, not sure if what device they're running but he
claimed it was the same issue.

-Brad
